vita3k/gui/src/game_selector.cpp

```diff
@@ -173,7 +217,7 @@ void draw_game_selector(GuiState &gui, HostState &host) {
                     if (host.cfg.show_game_background) {
                         if (gui.game_backgrounds.find(game.title_id) == gui.game_backgrounds.end())
                             load_game_background(gui, host, game.title_id);
-                        else
+                        else if (gui.current_background != gui.game_backgrounds[game.title_id])
```

Comment:
Oh my...

Reply:
not worry is perfect :)

Reply:
I will let it by for this pr but just know it frustrates me deeply.

Reply:
this is very good, it only change if needed.

Reply:
why fustrates ? get just better smart, spam changing not need is not logical :D

Reply:
If it was more efficient compiler would optimize it in...

Reply:
nothing to do with compiling it, it is about the execution of the programe, and about its function.

Reply:
The compiler's job is to make execution as fast and efficient as possible. It can add these extra instructions if it thinks it needs it, no need for you to do it manually (especially because it seems to be slower...)